src/lib/genesis_ledger_helper/genesis_ledger_helper.ml

```diff
@@ -228,6 +228,8 @@ module Ledger = struct
   let load_from_tar ?(genesis_dir = Cache_dir.autogen_path) ~logger
       ~(constraint_constants : Genesis_constants.Constraint_constants.t)
       ?accounts ~ledger_name_prefix ~expected_merkle_root filename =
+    (* TODO: pass in from above *)
+    let genesis_backing_type = Mina_ledger.Ledger.Root.Config.Stable_db in
```

Comment:
Looks like changes in this file need to be redo now that my PR is merged

Reply:
Should be resolved